docs/tutorial/1-buildingapp.rst

```diff
@@ -0,0 +1,124 @@
+==============
+Klein Tutorial
+==============
+
+In this tutorial we're going to build simple book library in Twisted Klein. Index
+page will list all available items. If user clicks
+on book link we'll download book text asynchronously and display result.
+
+Setting up
+==========
+
+Database for this tutorial is available in Klein github repo. 
+We'll assume you have requirements installed. Aside from Klein you also
+need to install Jinja2_ for templating and Twisted-Treq_
```

Comment:
Why are we using Jinja2 for templating rather than the built-in `twisted.web.template` templating?  Jinja2 doesn't support asynchrony.